it/JmsRequestReplyClientISpec.scala

```diff
@@ -26,9 +26,14 @@ import org.scalatest.matchers.must.Matchers
 import org.scalatest.{ BeforeAndAfterAll, FutureOutcome }
 import org.typelevel.log4cats.SelfAwareStructuredLogger
 import org.typelevel.log4cats.slf4j.Slf4jFactory
+import play.api.libs.json.Json
 import uk.gov.nationalarchives.omega.editorial.connectors._
-import uk.gov.nationalarchives.omega.editorial.services.EchoServer
+import uk.gov.nationalarchives.omega.editorial.editSets
+import uk.gov.nationalarchives.omega.editorial.models.GetEditSet
+import uk.gov.nationalarchives.omega.editorial.services.jms.{ ResponseBuilder, StubServer }
 
+import java.time.LocalDateTime
+import java.time.Month
 import scala.concurrent.duration.{ FiniteDuration, SECONDS }
 import scala.util.{ Failure, Success }
 
```

Comment:
With every such call (like getting an edit set), the "platform" (including the request-reply client) will implicitly be tested - eventually with a page-level integration test like `EditSetISpec`.

In the meantime, do you think it would to rename this, to be related to the specific call?

Reply:
Yes, I thought about this a bit - now that we are not going to test error handling here I think we are better off renaming this. I'll change it to `GetEditSetISpec`